app/templates/components/forms/profile-page.hbs

```diff
@@ -0,0 +1,34 @@
+<form class="ui large form" autocomplete="off" {{action 'submit' on='submit'}}>
+  <div class="ui stackable grid">
+    <div class="four wide column">
+      {{widgets/forms/image-upload
+        imageUrl=data.event.backgroundUrl
+        needsCropper=true
+        label=(t 'Event Image')
+        id='event_image'
+        icon='camera'
+        hint=(t 'Select Event Image')
+        maxSizeInKb=10000
+        helpText=(t 'We recommend using at least a 2160x1080px (2:1 ratio) image')}}
+    </div>
+    <div class="ten wide column">
```

Comment:
Make it a twelve wide column so that the page is symmetric. 

Semantic-ui is a 16 column grid system.